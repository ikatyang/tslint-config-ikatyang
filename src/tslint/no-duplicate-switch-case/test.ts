declare const string: string;

// good

switch (string) {
  case 'a':
  case 'b':
}

// bad

switch (string) {
  case 'a':
  case 'a':
  case 'b':
}
