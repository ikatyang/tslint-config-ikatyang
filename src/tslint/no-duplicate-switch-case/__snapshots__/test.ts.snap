// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lint 1`] = `
declare const string: string;

// good

switch (string) {
  case 'a':
  case 'b':
}

// bad

switch (string) {
  case 'a':
  case 'a':
       ~~~  [Duplicate switch case: ''a''.]
  case 'b':
}

`;
